Demo/Demo.xcodeproj/project.pbxproj

```diff
@@ -1058,7 +1058,7 @@
 			baseConfigurationReference = 73498B4265CA7D315E2FBF38 /* Pods-Demo.debug.xcconfig */;
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
-				CLANG_WARN_QUOTED_INCLUDE_IN_FRAMEWORK_HEADER = YES_ERROR;
+				CLANG_WARN_QUOTED_INCLUDE_IN_FRAMEWORK_HEADER = NO;
```

Comment:
I was getting the same error that the UI tests are getting ([example here](https://github.com/braintree/braintree_ios/actions/runs/4556828816/jobs/8040317812)) last week as well. This should resolve the issue but we should open a ticket with Cardinal to update their SDK to not use quoted includes as this would potentially be change merchants would have to make as well.

Posting the error for posterity since the CI run will be deleted after some time:
```
double-quoted include "<CARDINAL_CLASS_NAME>.h" in framework header, expected angle-bracketed instead [-Werror,-Wquoted-include-in-framework-header]
```